pkgs/by-name/in/invidious/package.nix

```
@@ -1,7 +1,7 @@
 {
   lib,
   callPackage,
-  crystal,
+  crystal_1_17,
```

```suggestion
  crystal_1_16,
```

Also needed to update this line